pom.xml

```diff
@@ -793,6 +794,12 @@
                 <artifactId>junit-platform-launcher</artifactId>
                 <version>${junit.platform.version}</version>
             </dependency>
+            <!-- transitive dep; override here to avoid buggy version -->
```

Comment:
Can you add which is using this as transitive dependency?

Reply:
Good point. We should probably remove these overrides at some point and see if they are still relevant?